cmd/substreams/run.go

```diff
@@ -124,6 +127,7 @@ func runRun(cmd *cobra.Command, args []string) error {
 		OutputModules:                       []string{outputModule}, //added for backwards compatibility, will be removed
 		ProductionMode:                      productionMode,
 		DebugInitialStoreSnapshotForModules: debugModulesInitialSnapshot,
+		Params:                              params,
```

Comment:
There wouldn't be `Params` on the `pbsubstreams.Request`. Its place is on `Module_Input_Params` (as it is already).